src/main/java/lotto/domain/Rank.java

```diff
@@ -0,0 +1,46 @@
+package lotto.domain;
+
+import lotto.domain.model.LottoNumber;
+import lotto.domain.model.Lotto;
+
+public enum Rank {
+    MISS(0, 0),
+    FIFTH(3, 5_000),
+    FOURTH(4, 5_0000),
+    THIRD(5, 1_500_000),
+    SECOND(5, 30_000_000),
+    FIRST(6, 2_000_000_000);
+    
+    private final int matchCount;
+    private final long winnerReturn;
+    
+    Rank(int matchCount, long winnerReturn) {
+        this.matchCount = matchCount;
+        this.winnerReturn = winnerReturn;
+    }
+    
+    public static Rank fromLottoNumber(int matchCount) {
```

Comment:
```suggestion
    public static Rank fromLottoNumber(int matchCount, boolean bonusMatch) {
```
이렇게 값이 전달되야 2등을 구할 수 있지 않을까?